airgun/entities/computeresource.py

```diff
@@ -75,6 +76,13 @@ def vm_poweroff(self, rhev_name, vm_name):
         if vm['Power'].widget.read() == 'On':
             vm['Actions'].widget.click(handle_alert=True)
 
+    def vm_import(self, cr_name, vm_name, hostgroup, location):
```

Comment:
Thanks for pointing this out, I've fixed it for "cr_name".
Because "rhev_name" is used incorrectly in this sense in multiple places unrelated to this PR, I'll fix that in another PR.